yat-docs/docusaurus.config.js

```diff
@@ -1,135 +1,159 @@
-                    {id: "nodejs", label: "Javascript / NodeJs"},
-                    {id: "kotlin", label: "Android / Kotlin"},
-                    {id: "swift5", label: "iOS / Swift 5"}
-                ],
-                // These options are passed directly to swagger-parser. Options are details at https://apitools.dev/swagger-parser/docs/options.html
-                swagger_parser: {
-                    validate: {schema: true}
-                },
-                openapi_generator: yaml.parse(fs.readFileSync("sdk_generator_opts.yml", "utf8"))
-            }
-        ]
-    ],
-    presets: [
-        [
-            '@docusaurus/preset-classic',
-            {
-                docs: {
-                    sidebarPath: require.resolve('./sidebars.js'),
-                    // Please change this to your repo.
-                    editUrl: undefined,
-                },
-                blog: {
-                    showReadingTime: true,
-                    // Please change this to your repo.
-                    editUrl:
-                        'https://github.com/facebook/docusaurus/edit/master/website/blog/',
-                },
-                theme: {
-                    customCss: require.resolve('./src/css/custom.css'),
-                },
-            },
-        ],
-    ],
-};
+const path = require('path');
+const yaml = require('yaml');
+const fs = require('fs');
+
+module.exports = {
+    title: 'Yat Developer Documentation',
+    tagline: 'Helping you build awesome stuff with 🖖',
+    url: 'https://docs.y.at',
+    baseUrl: '/',
+    onBrokenLinks: 'warn',
+    favicon: 'img/favicon.png',
+    organizationName: 'Yat Labs', // Usually your GitHub org/user name.
+    projectName: 'yat-docs', // Usually your repo name.
+    themeConfig: {
+        colorMode: {
+            defaultMode: 'dark',
+            disableSwitch: true
+        },
+        navbar: {
+            title: 'Yat Developer Documentation',
+            logo: {
+                alt: 'Yat',
+                src: 'img/yat-logo-white.svg',
+            },
+            items: [
+                {
+                    href: '#',
+                    label: 'Support',
+                    position: 'right',
+                },
+                {
+                    href: '#',
+                    label: 'APIs & SDKs',
+                    position: 'right',
+                },
+                {
+                    href: 'https://y.at',
+                    label: 'Y.at',
+                    position: 'right',
+                    target: '_blank',
+                },
+                {
+                    href: 'https://github.com/yat-labs/yat-docs',
+                    label: 'GitHub',
+                    position: 'right',
+                    target: '_blank',
+                },
+                {
+                    href: '#',
+                    label: 'Get API keys',
+                    position: 'right',
+					className: 'menuHighlight',

```

Comment:
Can you add the extra left margin as in the design so that there's a gap between this item and the others?